htmlelements-java/src/main/java/ru/yandex/qatools/htmlelements/loader/decorator/HtmlElementLocatorFactory.java

```diff
@@ -40,6 +43,32 @@ public ElementLocator createLocator(Field field) {
      * @param clazz Class for which locator will be created.
      */
     public ElementLocator createLocator(Class clazz) {
-        return new AjaxElementLocator(searchContext, new HtmlElementClassAnnotationsHandler(clazz));
+        return new AjaxElementLocator(searchContext, getTimeOut(clazz), new HtmlElementClassAnnotationsHandler(clazz));
+    }
+
+    protected int getTimeOut(Field field) {
```

Comment:
we should go here through all the super-classes until we found required annotation, the same way we look for @FindBy annotation [here](https://github.com/yandex-qatools/htmlelements/blob/master/htmlelements-java/src/main/java/ru/yandex/qatools/htmlelements/loader/decorator/HtmlElementFieldAnnotationsHandler.java#L65)